app/javascript/local/dropzone-cfg.js

```diff
@@ -113,4 +170,48 @@ function amendErrorLink (file, response) {
   let deleteButton = document.getElementById(file.upload.uuid).querySelector(".app-button--link")
   deleteButton.setAttribute("value", response.id)
 }
+
+function removeErrorMessages () {
+  // Clear error messages and hide error components from view when user uploads again
+  const errorSummary = document.querySelector('.govuk-error-summary')
+  errorSummary.querySelectorAll('li').forEach(listItem => {
+    listItem.remove()
+  })
+  errorSummary.classList.add('app-evidence-upload-hidden')
+
+  document.querySelector('.govuk-form-group').classList.remove('govuk-form-group--error')
+  document.querySelector('.govuk-error-message').classList.add('app-evidence-upload-hidden')
+  document.querySelector('.govuk-error-message').innerHTML = ""
+}
+
+function displayErrorMessage(errorSummary, errorMsg) {
+  // Display error to top of page
+  let pageDiv = document.querySelector('.govuk-grid-column-two-thirds')
```

Comment:
Uncomfortable using existing styling classnames as the selector - prefer using html `id` attributes or a dedicated `js-notification-banner`  to make the element selection more specific.